app/services/registration_csv_converter.rb

```diff
@@ -52,7 +52,7 @@ def convert_row(row)
     structural = {}
     structural[:isMemberOf] = [row['Collection']] if row['Collection']
     model_params[:structural] = structural
-    model_params[:access] = access(row['Rights']) if row.fetch('Rights') != 'default'
+    model_params[:access] = CocinaDroAccess.from_form_value(row['Rights']).value! if row.fetch('Rights') != 'default'
```

Comment:
Could you do this instead?
```suggestion
    model_params[:access] = CocinaDroAccess.from_form_value(row['Rights']).value_or(nil)
```